addons/payment/tests/common.py

```diff
@@ -6,20 +6,21 @@
 from lxml import objectify
 
 from odoo.fields import Command
-from odoo.tests.common import TransactionCase
+from odoo.addons.base.tests.common import BaseCommon
 from odoo.tools.misc import hmac as hmac_tool
 
 _logger = logging.getLogger(__name__)
 
 
-class PaymentCommon(TransactionCase):
+class PaymentCommon(BaseCommon):
 
     @classmethod
     def setUpClass(cls):
         super().setUpClass()
 
         cls.currency_euro = cls._prepare_currency('EUR')
         cls.currency_usd = cls._prepare_currency('USD')
+        cls._use_currency('EUR')
```

Comment:
Doesn't work since 3752b3166ec6bbb5c6ea55f96bb521381bf2b2d9
```suggestion
```

Reply:
Thanks for the reference. I just rebased and corrected tests.